docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+
+Once created, right click the revocation configuration and select `Edit properties`, go to the tab `Signing`, here you need to select `Enable NONCE extension support` then click `OK`.
+
+Verify that your OCSP is in the settings of the CA. Right click your CA, chose `Properties`. Navigate to the tab `Extension`, in the drop down list `Select extension` chose `Authority Information Access (AIA)`. Make sure that you have the following in the list of locations: `http://<fqdn of ndes server>/ocsp`, if you do not have it, add it via the button `Add...`. In this menu type the "http://" then insert "<ServerDNSName>" and type "/ocsp", the line you add should look like the following: `http://<ServerDNSName>/ocsp`, then validate by clicking `OK`. Also verify that `Include in the online certificate status protocol(OCSP) extension` is selected.
+
+By default OCSP has a two days delay to refresh his CRL information. Which mean if you revoke a certificate on MSPKI, it will take two days before PacketFence detect the certificate as revoked. If this delay is too long for your needs, you can change it on the NDES server. To do so, navigate to `Server Manager` > `Roles` > `Active Directory Certificate Service` and right click `Entreprise PKI`, in the menu select `Options...`. The delay can be changed by modifying the the value of: `Set CRL status to Expiring when expiring in:` to your convenience.
+
+
+RADIUS Certificate generation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the PKI of Microsft include that all your certificates will be deliver by the root CA of the PKI. The public key of this certificate is necessary on PacketFence. 
+
+As for RADIUS authentication we will need to generate a certificate for PacketFence.
+
+To generate the RADIUS certificate the template `WebServer` will be used.
+
+The idea now is to create the request (.CSR), a private key from the PacketFence server and submit the CSR to the NDES server. Connect to PacketFence via ssh and type the following in the CLI to generate the CSR and sign it with the private key:
+
+ openssl req -new -newkey rsa:2048 -nodes -keyout server.key -out server.csr
+
+You will be prompt for some information, here is an example of a valid configuration.
+
+* CN=packetfence.local
+* C=CA
+* ST=QC
+* Locality=Montreal
+* Organization=Inverse
+* Organization Unit=IT
+
+No fields are mandatory else than the CN.
+
+Once you have your CSR we will submit it to the NDES server.
+
+Display the content of the .csr file and open a web browser, reach the following:
```

Comment:
Don't take the user for a 5 year old child. No need to display the csr just tell him to paste the content in the field below